lib/core/jtrace.c

```diff
@@ -410,6 +431,10 @@ j_trace_init (gchar const* name)
 		return;
 	}
 
+	if (j_trace_flags & J_TRACE_COMBINED)
+	{
+		j_trace_combined_timers = g_hash_table_new_full(g_str_hash, g_str_equal, g_free, g_free);
+	}
```

Comment:
This has inconsistent formatting (we want empty lines before and after if/while/etc.).

Reply:
ok